ratelimit/options.go

```diff
@@ -0,0 +1,16 @@
+// See LICENSE for licensing terms.
+
+package ratelimit
+
+import "time"
+
+const infinityDuration time.Duration = 0x7fffffffffffffff
+
+type Option func(*rateLimiter)
+
+// WithRateLimiter customizes your limiter in the middleware
```

Comment:
can this be a full sentence ... how does it customize? It does give much info to the consumer.